### PR TITLE
Add alpine 3.18

### LINKED ISSUE #37396: Alpine 3.18 is supported
### Type of issue

Missing information

### Description

Alpine 3.18 includes long awaited DNS TCP fallback via libmusl 1.2.4 as noted here.
https://www.alpinelinux.org/posts/Alpine-3.18.0-released.html

Alpine 3.18 is supported in the .NET Docker repo: 
https://github.com/dotnet/dotnet-docker/tree/main/src/sdk/6.0/alpine3.18

The "Supported distributions" table should mention 3.18 but only mentions up to 3.17

image

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/install/linux-alpine#supported-distributions

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/install/linux-alpine.md

### Document Version Independent Id

f066a748-0a7d-7506-3b18-c422fef13293

### Article author

adegeo

---
[Associated WorkItem - 169712](https://dev.azure.com/msft-skilling/Content/_workitems/edit/169712)

### PR DESCRIPTION
**Pull request by adegeo**
- Rewrote table columns a bit to include another alpine version
- Rewrote the architectures table.
  I used the alpine package release to figure this out, but I'm not sure if that's the best source?? https://pkgs.alpinelinux.org/packages?branch=edge&page=2&name=dotnet%2a

  @ayakael can you confirm this information? I'm not sure where to source the info.

Fixes #37396

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/linux-alpine.md](https://github.com/dotnet/docs/blob/7332c67749e7695017bd64457a548b2ee3e37577/docs/core/install/linux-alpine.md) | [Install the .NET SDK or the .NET Runtime on Alpine](https://review.learn.microsoft.com/en-us/dotnet/core/install/linux-alpine?branch=pr-en-us-37526) |

### CODE REVIEW

**Comment by svick** [comment]
@adegeo Sorry, I don't think I'll be able to review this anytime soon.

**Comment by adegeo** [comment]
We can push this through for now since it will be updated for .NET 8 and require another review.

**Comment by adegeo** [comment]
@tdykstra Please review when you can, thanks!

File: docs/core/install/linux-alpine.md [hunk 1]
```diff
@@ -83,7 +84,7 @@ apk add bash icu-libs krb5-libs libgcc libintl libssl1.1 libstdc++ zlib
 
 [!INCLUDE [linux-libgdiplus-general](includes/linux-libgdiplus-general.md)]
 
-To install libgdiplus on Alpine 3.16 or newer (older versions don't include the package), run:
+To install libgdiplus on Alpine 3.16 or newer, run:
```
Comment (tdykstra) [inline docs/core/install/linux-alpine.md:87]:
```suggestion
To install libgdiplus on Alpine 3.16 or later, run:
```

**Review by tdykstra — APPROVED** [review]
LGTM, 1 nit

**Comment by adegeo** [comment]
@tdykstra done!

**Pull request merged**